tests/snapshots/files__block-diamond-optimize.snap

```diff
@@ -10,28 +10,24 @@ expression: visualization.result
   c5_: int = const 0;
```

Comment:
I didn't look closely at any of these snapshot diffs. Should I ?

Reply:
Probably not. I skimmed them to make sure that this PR doesn't add a ton of code for some reason, but I see lots of the small examples getting their if statements deleted so I'm pretty sure it's working.